### PR TITLE
Added alt text

### PR DESCRIPTION
**Pull request by MeeraDi**

### CODE REVIEW

**Comment by MeeraDi** [comment]
@Rick-Anderson, @mcleblanc, @v-andreaco
This PR addresses missing alt text and is ready for final review.

**Comment by MeeraDi** [comment]
@Rick-Anderson, @mcleblanc, @v-andreaco
This PR addresses missing alt text and is ready for final review.

File: iis/application-frameworks/running-php-applications-on-iis/enable-php-syntax-highlighting-on-iis-7-and-above.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ by Tali Smith
 
 You can configure Internet Information Services 7 (IIS 7) and above to output syntax-highlighted source code for PHP files stored on the Web server or Web site. This feature is useful in development environments because you can quickly make the source code accessible to other team members.
 
-![](enable-php-syntax-highlighting-on-iis-7-and-above/_static/image1.gif) Note that you should not attempt these procedures on any Internet-accessible Web site as this greatly compromises the security of your Web application.
+:::image type="icon" source="enable-php-syntax-highlighting-on-iis-7-and-above/_static/image1.gif" ::: Note that you should not attempt these procedures on any Internet-accessible Web site as this greatly compromises the security of your Web application.
```
Comment (Rick-Anderson) [inline iis/application-frameworks/running-php-applications-on-iis/enable-php-syntax-highlighting-on-iis-7-and-above.md:16]:
```suggestion
> [!NOTE]
> You should not attempt these procedures on any Internet-accessible Web site as this greatly compromises the security of your Web application.
```

Comment (Rick-Anderson) [reply iis/application-frameworks/running-php-applications-on-iis/enable-php-syntax-highlighting-on-iis-7-and-above.md:16]: Let's do notes this way when possible.

**Review by Rick-Anderson — APPROVED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**